src/Compilers/CSharp/Portable/BoundTree/NullabilityRewriter.cs

```diff
@@ -162,6 +162,25 @@ Symbol remapLocal(SourceLocalSymbol local)
             }
         }
 
+        public override BoundNode? VisitIndexOrRangePatternIndexerAccess(BoundIndexOrRangePatternIndexerAccess node)
+        {
+            BoundExpression argument = (BoundExpression)this.Visit(node.Argument);
+            BoundExpression lengthOrCountAccess = node.LengthOrCountAccess;
```

Comment:
>ode.LengthOrCountAccess [](http://example.com/codeflow?start=51&length=23)

Is this node never exposed via a public API?

Reply:
You can get the Length/Count property via IOperation

Reply:
Hm, I see the problem ;-)

Reply:
Added test and filed  [https://github.com/dotnet/roslyn/issues/57884](https://github.com/dotnet/roslyn/issues/57884)